src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md

```diff
@@ -0,0 +1,57 @@
+
+## The basic way to choose image types
+
+For the past few years, the easiest and most straightforward way to choose your image type was between photographs and graphics. As a quick rule of thumb: **photographs should be JPEGs and everything else should be PNGs**. There are two exceptions:
+
+1. If you are using an animated image (or video converted to image), definitely use a GIF
+2. If you're using images like icons (where they need to scale with your typography), see if you can convert that image to an SVG (Sketch is easily the best at [converting to SVG](https://medium.com/sketch-app-sources/the-best-way-to-export-an-svg-from-sketch-dd8c66bb6ef2))
+
+There are a bunch of other image formats, such as BMPs, EXIFs, and TIFFs, but those have largely fallen out of favor even though they still technically work.
+
+With this advice, you can't go wrong. But the web is still evolving and there are new file formats to consider when putting your images on the web.
+
+## The advanced way to choose image types
+
+Recently there has been a rise in next-generation image formats to make the web lighter and more accessible. Unfortunately, the rules for choosing these formats is not as straightforward as it once was. Here are the 3 main types of next-gen image formats you should be considering when uploading your next image onto Shubox.
+
+### JPEG 2000
+
+[JPEG 2000](https://jpeg.org/jpeg2000/index.html) is a newer incarnation of the JPEG format. The primary purpose of the format is to preserve metadata through radical compression techniques. While most traditional JPEGs have a limited set of metadata for photographs, JPEG 2000 uses structured XML to include a vast array of metadata information.
+
+This is useful for applications like film and medical imaging because a great deal of data needs to be associated with the image. For example, film stills need to store information about high dynamic range and different color spaces. Medical imaging requires absolute precision in image clarity in spite of the large volume of data. With JPEG 2000, you can encode these files completely losslessly, with good-enough compression performance to make transfer acceptable.
+
+Even if you aren't in the film or healthcare industry, you can take advantage of these new compression techniques in your current JPGs by switching to JPEG 2000. There is one caveat: if you are writing for the web, JPEG 2000 is only [supported on Safari](https://caniuse.com/#feat=jpeg2000), so it will only benefit your customers on Macs, iPhones, and iPads. As long as you use the other two formats that follow, you should definitely consider JPEG 2000 in your arsenal of image formats.
```

Comment:
What are the benefits to using JPEG 2000 outside of film/healthcare. Are the raw compression gains compelling?

Reply:
as I pointed out in the post, you need it because there is no support for the other next-gen formats in Safari

Reply:
I get that part -- it's supported by Safari.  But, if your images don't need all that extra metadata support and they're just "dumb" images, are the associated compression gains significant enough to convince someone to use this as their format of choice?

TLDR: Apples to apples, Is this better than the same vanilla JPEG version? By how much?